rpcs3/Emu/Cell/PPUThread.cpp

```diff
@@ -424,19 +440,31 @@ std::string ppu_thread::dump() const
 	fmt::append(ret, "XER = [CA=%u | OV=%u | SO=%u | CNT=%u]\n", xer.ca, xer.ov, xer.so, xer.cnt);
 	fmt::append(ret, "VSCR = [SAT=%u | NJ=%u]\n", sat, nj);
 	fmt::append(ret, "FPSCR = [FL=%u | FG=%u | FE=%u | FU=%u]\n", fpscr.fl, fpscr.fg, fpscr.fe, fpscr.fu);
-	fmt::append(ret, "\nCall stack:\n=========\n0x%08x (0x0) called\n", cia);
 
-	//std::shared_lock rlock(vm::g_mutex); // Needs optimizations
```

Comment:
Dont uncomment this.

Reply:
oh, remove it you mean? rebasing annoyance again, i'll put it back.